### DISCUSSION TITLE
Pick colour to use as mask?

### DISCUSSION THREAD

**Discussion by Mige5** [lllyasviel/Fooocus#3793] [General]
Hey, I dont see there being an option to just pick a colour to select an area for inpaint, kinda like how "magic wand" works in photoshop?
- would make stuff like this much faster than having to manually paint the area for each image. - I dont mean generating the mask the way there already is options, but I mean like it would select the area the same way as if I had manually painted over to select it, so it would also allow me to manually paint more areas if I wanted to.
![testi](https://github.com/user-attachments/assets/fa86f346-fb12-4394-81e3-d13a9b5bdd99)
![testi2](https://github.com/user-attachments/assets/ce8cb4af-730a-4728-bccf-29833e84b510)

**Comment by ghost** [comment]
Do it in photoshop or another program and add the mask in Fooocus separately (`Inpaint or Outpaint -> Enable Advanced Masking Features -> Mask Upload`).
